core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -71,6 +72,8 @@
   UNDERFS_OSS_CONNECT_TIMEOUT(Name.UNDERFS_OSS_CONNECT_TIMEOUT),
   UNDERFS_OSS_CONNECT_TTL(Name.UNDERFS_OSS_CONNECT_TTL),
   UNDERFS_OSS_SOCKET_TIMEOUT(Name.UNDERFS_OSS_SOCKET_TIMEOUT),
+  UNDERFS_S3_CANONICAL_USER_ID_TO_USERNAME_MAPPING(
```

Comment:
Done, I thought CANONICAL_USER is more descriptive so that the user will not take the default account ID (12 digits in AWS)...  It's fine because we explicitly mentioned how to find CANONICAL_USER_ID in the docs.